README.md

```diff
@@ -54,36 +54,55 @@ This will:
 
 Then the tests must be compiled with `CFG_GP_PACKAGE_PATH=<path>`.
 
+### Copying xtest TAs to the target
+
+xtest builds an executable `xtest`, and several .ta Trusted Applications.
+
+These need to be copied into the target rootfs.
+
+File|OS|Destination
+---|---|---
+xtest|Linux|/bin
+"|Android|/system/bin
+*.ta|Linux|/lib/optee_armtz/
+"|Android|/system/lib/optee_armtz/
+
+For example, for Linux:
+
+```
+ $ scp ./host/xtest/build/xtest/xtest root@$MY_IP:/bin
+ $ scp `find . -name *.ta` root@$MY_IP:/lib/optee_armtz
+```
 
 ### HOWTO run xtest
 
 	# all xtest
 	boot and execute on your target
-	$ modprobe optee_armtz
+	$ modprobe optee
```

Comment:
It could be worth mentioning that this step is only needed if the driver is compiled as a loadable module. For easy testing etc I guess it's more or less assumed that the driver is compiled into the kernel.